src/logic/outfits.ts

```diff
@@ -0,0 +1,61 @@
+import { Outfits } from '@dcl/schemas'
+import { createNamesOwnershipChecker } from '../ports/ownership-checker/names-ownership-checker'
+import { createWearablesOwnershipChecker } from '../ports/ownership-checker/wearables-ownership-checker'
+import { AppComponents, TypedEntity } from '../types'
+
+export async function getOutfits(
+  components: Pick<AppComponents, 'metrics' | 'content' | 'theGraph' | 'config' | 'fetch' | 'ownershipCaches'>,
+  ethAddress: string
+): Promise<TypedEntity<Outfits> | undefined> {
+  const outfitsEntities: TypedEntity<Outfits>[] = await components.content.fetchEntitiesByPointers([ethAddress])
+
+  if (!outfitsEntities || outfitsEntities.length === 0) {
+    return undefined
+  }
+
+  const outfitsEntity = outfitsEntities[0]
+
+  if (!outfitsEntity.metadata || outfitsEntity.metadata.outfits.length === 0) {
+    return outfitsEntities[0]
+  }
+
+  const outfits = outfitsEntities[0].metadata as Outfits
+
+  const wearablesOwnershipChecker = createWearablesOwnershipChecker(components)
```

Comment:
this looks like a component, why it's created here?

Reply:
Looks like but it's not. It's like a disposable object.